seewav.py

```diff
@@ -172,41 +181,60 @@ def visualize(audio,
     `time` amount of audio shown at once on a frame.
     `oversample` higher values will lead to more frequent changes.
     `fg_color` is the rgb color to use for the foreground.
+    `fg_color2` is the rgb color to use for the second wav if stereo is set.
     `bg_color` is the rgb color to use for the background.
     `size` is the `(width, height)` in pixels to generate.
+    `stereo` is whether to create 2 waves.
     """
     try:
         wav, sr = read_audio(audio, seek=seek, duration=duration)
     except (IOError, ValueError) as err:
         fatal(err)
         raise
-    wav = wav.mean(0)
-    wav /= wav.std()
+    # wavs is a list of wav over channels
+    wavs = []
+    if not stereo:
```

Comment:
nit: I usually prefer `if stereo: ... else:`, e.g. if the two branches of `if: ... else: ...` exist, then it is better to have no negation in the if, by exchanging the two branches. This is because the `else:` otherwise means `if not (not stereo)`, eg. double negation, which is more complex to parse for the brain.